lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :menu , :basket
+
+  def initialize
+    @menu = Menu.new
+    @basket = {}
+  end
+
+  def add(item, quantity)
+    basket[item] = quantity
+    "#{quantity} - #{item} has been added to your basket"
+  end
+
+  def price(item, quantity)
+    price = quantity * menu.dishes[item].to_i
+  end
+
+  def calculate_price
+    basket.map { |item, quantity| price(item, quantity).to_i }.inject(:+)
+  end
+
+  def overview
+    return 'Basket is empty' if basket.empty?
+    basket.map { |item, quantity| "#{item} x#{quantity}: £#{price(item, quantity)}... "}
+      .join("\n") + total
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)